lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,168 @@ fields:
     example: |
       {
         "id": "SH-12508041"
-      }
\ No newline at end of file
+      }
+
+  - name: questionnaire
+    required: 'Yes'
+    type: Questionnaire
+    url: http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire
+    description: The Questionnaire that defines and organizes the questions for which answers are being provided.
+    action: update
+    example: |
+      {
+        "questionnaire": "https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041"
+      }
+
+  - name: status
```

Comment:
maybe mention it must be in progress?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/cf9610665943eeeb9f8a443ab9e3814f9f51a0cd

Reply:
can we use the note field
https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/r4/encounter.yaml#L14

Reply:
https://github.com/cerner/fhir.cerner.com/pull/669/commits/ab22d344ac9b670198de9cbbe535af9a1b675d52